src/aap_eda/settings/core.py

```diff
@@ -141,7 +152,12 @@
 DEFAULT_WORKER_HEARTBEAT_TIMEOUT = 60
 DEFAULT_WORKER_TTL = 5
 
-RQ_STARTUP_JOBS = []
+RQ_STARTUP_JOBS = [
+    {
+        "func": "aap_eda.tasks.analytics.schedule_gather_analytics",
+        "job_id": "start_analytics_scheduler",
+    },
+]
```

Comment:
This should only be set if the analytics flag is enabled, see - https://github.com/ansible/eda-server/commit/fd0b510fe0d498a9324fb66e4aaceb6cad0300f9#diff-2b11636ac9ca504baca6396a8dac9ffd1684853f73bc628da021fb2582bb6b22L156

Reply:
it is guarded by the feature flag in aap_eda.tasks.analytics.schedule_gather_analytics. In core.py the feature flag settings have not been loaded.

Reply:
Great point, that sounds good, thanks Bill